style.css

```diff
@@ -558,3 +572,431 @@ ul li {
+  margin-bottom: 5px;
+}
+
+@media only screen and (min-width: 768px) {
+  body {
+    width: 100%;
+  }
+
+  .top-nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 16px;
+  }
+
+  .top-nav-ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .top-nav-a a {
+    padding: 10px;
+    text-decoration: none;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-size: 16px;
+    font-weight: 600;
+    color: rgb(9, 30, 66);
+  }
+
+  .greating {
+    display: none;
+  }
+
+  .menu {
+    display: none;
+    width: 0;
+    height: 0;
+  }
+
+  header {
+    width: 100%;
+    background-image: url(./assest/Header-llustration-desktop@2x.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    justify-items: center;
+  }
+
+  .main .mobile-h1 {
+    display: none;
+  }
+
+  .main .desk-h1 {
+    display: flex;
+    flex-direction: column;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    color: rgb(23, 43, 77);
+    margin-top: 50px;
+    align-items: center;
+  }
+
+  .support-text {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: rgb(23, 43, 77);
+    width: 60%;
+    margin-top: 24px;
+  }
+
+  .socialicon {
+    margin-top: 30px;
+    margin-right: 30px;
+  }
+
+  .arrow {
+    margin-bottom: 200px;
+    margin-right: 30px;
+  }
+
+  .recentwork .indicator {
+    display: none;
+   }
+
+  .text h2 {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: rgb(23, 43, 77);
+    }
+
+  section .recentwork {
+   display: flex;
+   flex-direction: column;
+   align-items: center;
+   width: 100%;
+   height: 1200px;
+  }
+
+  .card-work {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 24px;
+    margin-left: 134px;
+    margin-right: 134px;
+    margin-top: 52px;
+  }
+
+  .card-title {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: rgb(23, 43, 77);
+ }
+
+ .tag {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 8px;
+
+ }
+
+ .tag a {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: rgb(23, 43, 77);
+ }
+
+ .about {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: wrap;
+  margin-left: 259px;
+  margin-right: 259px;
+  margin-top: 192px;
```

Comment:
- [ ] Kindly decrease margin-top value here to match the design in figma. 
![image](https://user-images.githubusercontent.com/91486791/186754361-492a3784-50c7-480d-9d35-229295d9e8c9.png)